wremnants/datasets/datagroups.py

```diff
@@ -200,14 +201,15 @@ def setHists(self, baseName, syst, procsToRead=None, label=None, nominalIfMissin
             
         for procName in procsToReadSort:
             logger.debug(f"Reading group {procName}")
+            
             if procName not in self.groups.keys():
                 raise RuntimeError(f"Group {procName} not known. Defined groups are {list(self.groups.keys())}.")
             group = self.groups[procName]
             group.hists[label] = None
 
             for i, member in enumerate(group.members):
                 if procName == nameFake and member.name in fakesMembersWithSyst:
```

Comment:
Shouldn't nameFake be a member of the class? It doesn't even need a set function, but this would be an improvement

Reply:
Yes, this would be good. Eventually the name must be consistent with what was originally set in the sample dictionary in datasetDict_v9.py, then setupCombineWmass also had an option to choose a different name to be used as Fakes (which is then set in CardTool, in case there could be multiple definitions for tests or something else).
So here I had originally set it to be a variable under the assumption this could change and not always be "Fake", but at the moment the consistency with setupCombineWmass and CardTool is not yet guaranteed

Reply:
But I think it is not in the scope of this PR